docs/reference/reference-interface/job-config-reference.rst

```diff
@@ -164,3 +164,5 @@ The following configuration settings are supported:
 
 -  ``slurm``: Slurm cluster details may optionally be specified as for :ref:`experiments
    <slurm-config>`.
+
+-  ``pbs``: PBS cluster details may optionally be specified as for :ref:`experiments <pbs-config>`.
```

Comment:
The "as for" doesn't sound right, although I can't see what actually comes after that once it's replaced.  Is "as for" the same thing as "as per" or "as described in"?  If so, those may be a better choice of wording.